R/cedta.R

```diff
@@ -29,6 +29,9 @@ cedta = function(n=2L) {
   if (isTRUE(env$.datatable.aware)) {  # dtplyr 184
     return(TRUE)
   }
+  if (isFALSE(env$.datatable.aware)) {  #5654
```

Comment:
We have `isTRUEorFALSE`:

https://github.com/Rdatatable/data.table/blob/cde7333938a590a3cddbda1b02103650e2f55d15/R/utils.R#L14

So we could more concisely write:

```r
if (isTRUEorFALSE(env$.datatable.aware)) return(env$.datatable.aware)
```